### DISCUSSION TITLE
Allow Adding Item Photos, Manuals, etc. from URL

### DISCUSSION THREAD

**Discussion by Fumanchooo** [sysadminsmedia/homebox#741] [Ideas]
### What is the problem you are trying to solve with this feature?

I typically add product photos and manuals from the manufacturer website or store listing when I add new items to Homebox.  The current process requires the user to download the files locally and then upload them to Homebox.

### What is the solution you are proposing?

Allowing the user to enter a URL instead of selecting a file and then having Homebox retrieve the file from the source would save a step for this use case.

### Contributions

- [x] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [ ] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [ ] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by tankerkiller125** [comment]
Please note we're converting this issue into a discussion as per our new procedures/policies. You can read more about that in our announcement #712

**Reply by agryson** [reply]
And/or support pasting from the clipboard in the asset upload widget?

**Comment by agryson** [comment]
Relevant discussion: https://github.com/sysadminsmedia/homebox/discussions/721
